### PR TITLE
docs: Adding Router service hierarchical injection level

### PR DESCRIPTION
**Pull request by krp12345**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Each service in angular framework should have a level at which it is provided and who provides and when. This is for better clarity. 

Issue Number: N/A

## What is the new behavior?
Router service will have level of hierarchy at which it is provided.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/router/src/router.ts [hunk 1]
```diff
@@ -87,7 +87,8 @@ export const subsetMatchOptions: IsActiveMatchOptions = {
 /**
  * @description
  *
- * A service that provides navigation among views and URL manipulation capabilities.
+ * A service that facilitates navigation among views and URL manipulation capabilities.
+ * This service is provided at root level in hierarchical dependency injection structure by importing module returned from `forRoot` method of [RouterModule](api/router/RouterModule) .
```
Comment (atscott) [inline packages/router/src/router.ts:91]:
```suggestion
 * This service is provided in the root scope and configured with [provideRouter](api/router/provideRouter).
```

File: packages/router/src/router.ts [hunk 2]
```diff
@@ -87,7 +87,8 @@ export const subsetMatchOptions: IsActiveMatchOptions = {
...
  *
  * @see {@link Route}
```
Comment (atscott) [inline packages/router/src/router.ts:93]:
```suggestion
 * @see {@link Route}
 * @see {@link provideRouter}
```

**Review by atscott — CHANGES_REQUESTED** [review]

**Review by krp12345 — COMMENTED** [review]
Agreed with @atscott

**Comment by krp12345** [comment]
@atscott I did not know that new way to configure routes. Thanks

**Review by atscott — APPROVED** [review]

**Comment by atscott** [comment]
This PR was merged into the repository by commit 13bc8d7d81ae1c8a5281522e9c8f4c03ed07fa5f.

The changes were merged into the following branches: main, 18.1.x

**Pull request closed without merging**